fastlane/Fastfile

```diff
@@ -0,0 +1,30 @@
+
+default_platform(:android)
+
+platform :android do
+  lane :deploy_production do
+    increment_version_code(
```

Comment:
Just curious, what about the version name?

Reply:
I don't found a Fastlane documentation about it.